lib/logitech_receiver/base.py

```diff
@@ -44,6 +44,14 @@
 _MEDIUM_MESSAGE_SIZE = 15
 _MAX_READ_SIZE = 32
 
+# mapping from report_id to message length
+report_length = {
+	0x10: _SHORT_MESSAGE_SIZE,
+	0x11: _LONG_MESSAGE_SIZE,
+	0x20: _MEDIUM_MESSAGE_SIZE,
```

Comment:
This is missing a length (0x21). The names are:
- 0x10 short HID++ report
- 0x11 long HID++ report
- 0x20 short DJ report
- 0x21 long DJ report

https://lekensteyn.nl/files/logitech/Unifying_receiver_DJ_collection_specification_draft.pdf#page=5
(a copy of this doc can also be found in the Solaar docs folder)

Reply:
OK.  I was just taking existing code and cleaning it up a bit, but I'll add that one.

Reply:
Done